### PR TITLE
fixed comments warning in ast

### PR DESCRIPTION
**Pull request by zhexuany**
This PR aims to address comments warning produced by `goword`. There are some comments in form of `foo ...` means I do not know how to write comments for it. Please check it out and give me some advice.

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: ast/misc.go [hunk 1]
```diff
@@ -318,10 +318,12 @@ func (n *FlushStmt) Accept(v Visitor) (Node, bool) {
 type KillStmt struct {
 	stmtNode
 
+	// Query ...
```
Comment (shenli) [inline ast/misc.go]: Query indicates whether terminate a single query on this connection or the whole connection.

File: ast/misc.go [hunk 2]
```diff
@@ -318,10 +318,12 @@ func (n *FlushStmt) Accept(v Visitor) (Node, bool) {
...
 	// If Query is true, terminates the statement the connection is currently executing, but leaves the connection itself intact.
 	// If Query is false, terminates the connection associated with the given ConnectionID, after terminating any statement the connection is executing.
 	Query        bool
 	ConnectionID uint64
+	// TiDBExtension ...
```
Comment (shenli) [inline ast/misc.go]: TiDBExtension is used to indicate whether the user knows he is sending kill statement to the right tidb-server.

File: ast/misc.go [hunk 3]
```diff
@@ -701,6 +703,7 @@ type SelectStmtOpts struct {
 // TableOptimizerHint is Table level optimizer hint
 type TableOptimizerHint struct {
 	node
+	// HintName ...
```
Comment (shenli) [inline ast/misc.go]: HintName is the name or alias of the table(s) which the hint will affect.

**Comment by zhexuany** [comment]
@shenli DONE. PTAL.

**Pull request merged**